### PR TITLE
move build-necessary packages to prod packages

### PR DESCRIPTION
**Pull request by WCRest**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,13 +32,17 @@
...
     "install": "^0.8.2",
     "ipware": "^1.0.0",
     "mongoose": "^4.7.6",
+    "ng-annotate-webpack-plugin": "^0.2.1-pre",
     "ng-map-autocomplete": "0.0.1",
     "ngclipboard": "^1.1.1",
     "nodemailer": "^4.0.1",
     "nodemailer-express-handlebars": "^2.0.0",
+    "node-sass": "^4.5.2",
+    "node-sass-middleware": "^0.11.0",
```
Comment (tjgupta) [inline package.json]: This one shouldn't be needed for the build I don't think.

File: package.json [hunk 2]
```diff
@@ -45,26 +53,18 @@
     "shortid": "^2.2.6",
     "start": "^4.0.2",
     "sunlight-congress-api": "^0.3.1",
-    "ua-parser": "^0.3.5"
+    "ua-parser": "^0.3.5",
+    "webpack": "^2.3.2",
+    "webpack-dev-middleware": "^1.10.1"
```
Comment (tjgupta) [inline package.json]: This one also shouldn't be needed for build. Only developer convenience.

File: package.json [hunk 3]
```diff
@@ -45,26 +53,18 @@
...
     "eslint-plugin-standard": "^2.1.1",
-    "html-loader": "^0.4.5",
-    "ng-annotate-webpack-plugin": "^0.2.1-pre",
-    "node-sass": "^4.5.2",
-    "node-sass-middleware": "^0.11.0",
     "nodemon": "^1.11.0",
-    "uglifyjs-webpack-plugin": "^0.4.1",
-    "webpack": "^2.3.2",
-    "webpack-dev-middleware": "^1.10.1"
+    "uglifyjs-webpack-plugin": "^0.4.1"
```
Comment (tjgupta) [inline package.json]: This one probably IS needed for the build.

**Comment by WCRest** [comment]
changes made, including the issue with static assets not working with pm2

**Pull request merged**